docs/sources/vendor/PaloaltoNetworks/panos.md

```diff
@@ -32,7 +32,7 @@
 | key                     | sourcetype     | index          | notes          |
 |-------------------------|----------------|----------------|----------------|
 | pan_panos_log           | pan:log       | netops          | none          |
-| pan_panos_globalprotect | pan:pan_globalprotect | netfw | none |
+| pan_panos_globalprotect | pan:globalprotect | netfw | none |
```

Comment:
Please, don't forget to also add [here](https://github.com/splunk/splunk-connect-for-syslog/blob/b915238328521657759b2748c37006e4e01e4616/docs/sources/vendor/PaloaltoNetworks/panos.md?plain=1#L21)

Reply:
done! ;)